.github/workflows/lint.yml

```diff
@@ -24,7 +24,16 @@ jobs:
             .ts
             .js
 
+      - name: install node v12
+        uses: actions/setup-node@v1
+        with:
+          node-version: 12
```

Comment:
Should we use Node `16` instead since it's the [active long-term support](https://nodejs.org/en/about/releases/) version?